.circleci/config.yml

```diff
@@ -32,3 +34,21 @@ jobs:
       - run:
           name: Install
           command: rake install
+      - persist_to_workspace:
+          root: .
+          paths: [ bin/liche, Dockerfile.test ]
+
+  docker:
+    docker:
+      - image: docker:17.05.0-ce-git
+    steps:
+      - attach_workspace: { at: . }
+      - setup_remote_docker
+      - run: docker build -f Dockerfile.test -t raviqqe/docker-liche:latest .
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build
+      - docker: { requires: [ build ] }
```

Comment:
To see CI results on this PR, you'd need to switch on "Build forked pull requests" in your CircleCI advanced settings: https://circleci.com/gh/raviqqe/liche/edit#advanced-settings

<img width="745" alt="screen shot 2018-11-19 at 14 57 51" src="https://user-images.githubusercontent.com/3473798/48714912-a57c9200-ec0b-11e8-96a1-0277848e4396.png">

In the meantime, here's the CI working on my fork: https://circleci.com/gh/iamdanfox/liche